twoliter/embedded/Makefile.toml

```diff
@@ -315,15 +315,25 @@ dependencies = ["setup-build"]
 script_runner = "bash"
 script = [
 '''
+SDK_PLATFORM="$(docker version --format '{{.Server.Os}}/{{.Server.Arch}}')"
+KRANE="${TWOLITER_TOOLS_DIR}/krane"
+
 if [ ! -s "${BUILDSYS_EXTERNAL_KITS_DIR}/.sdk-verified" ]; then
    echo "Twoliter could not validate '${TLPRIVATE_SDK_IMAGE}', refusing to continue" >&2
    exit 1
 fi
 if ! docker image inspect "${TLPRIVATE_SDK_IMAGE}" >/dev/null 2>&1 ; then
-  if ! docker pull "${TLPRIVATE_SDK_IMAGE}" ; then
+  echo "Pulling SDK '${TLPRIVATE_SDK_IMAGE}'"
+  ${KRANE} pull "${TLPRIVATE_SDK_IMAGE}" /dev/stdout --platform "${SDK_PLATFORM}" \
+    | docker load
```

Comment:
Not sure if it applies here, but there's potential for a performance hit writing to a pipe vs. a seekable file, if `krane` would otherwise be able to transfer parts in parallel.

This is fine, but I would probably write it to a temp file under the build directory, and register a cleanup function, vs. relying on `/dev/stdout`. Then there'd be clearer separation between "failed to pull" and "failed to load".